src/groovy/org/pih/warehouse/api/NotificationType.groovy

```diff
@@ -0,0 +1,14 @@
+package org.pih.warehouse.api
+
+enum NotificationType {
```

Comment:
:) yeah this doesn't feel right to me

Reply:
this might be an ok solution, but I don't even think I understand the problem. Could someone upload a video to the ticket that demonstrates the problem and all of the places it occurs? Are we basically just trying to solve a communication issue between GSP and React? Or is there something else going on?

Reply:
So I reviewed the rest of the code and I think understand the problem a little better. It does seem to be a "how do we get some bits of information from the previous request to the next request within the confines of a redirect". Which for grails / spring is the flash scope. But I've never had to think about this from Reacts perspective. 

I'm less hesitant about moving forward with this solution so I'll leave it up to you guys.